docs/features.md

```diff
@@ -36,20 +36,20 @@ Feature                                | ID       | Status             | Notes
 `DEVICE_RESET`                         | `0x1802` | Unsupported        |
 `OOBSTATE`                             | `0x1805` | Unsupported        |
 `CONFIG_DEVICE_PROPS`                  | `0x1806` | Unsupported        |
-`CHANGE_HOST`                          | `0x1814` | Supported          |
+`CHANGE_HOST`                          | `0x1814` | Supported          | `_feature_change_host`
 `HOSTS_INFO`                           | `0x1815` | Partial Support    | `get_host_names`, partial listing only
 `BACKLIGHT`                            | `0x1981` | Unsupported        |
 `BACKLIGHT2`                           | `0x1982` | Supported          | `_feature_backlight2`
 `BACKLIGHT3`                           | `0x1983` | Unsupported        |
 `PRESENTER_CONTROL`                    | `0x1A00` | Unsupported        |
 `SENSOR_3D`                            | `0x1A01` | Unsupported        |
-`REPROG_CONTROLS`                      | `0x1B00` | Listing Only       | `get_keys`, only listing
-`REPROG_CONTROLS_V2`                   | `0x1B01` | Unsupported        |
+`REPROG_CONTROLS`                      | `0x1B00` | Unsupported        | `get_keys`, only listing
```

Comment:
It is right that ‘get_keys’ note is behind the ‘unsupported’?

Reply:
No, that should be moved to V2